pilot/pkg/security/authn/v1beta1/policy_applier_test.go

```diff
@@ -1536,6 +1536,9 @@ func TestInboundMTLSSettings(t *testing.T) {
 	}
 
 	testNode := &model.Proxy{
+		Labels: map[string]string{
```

Comment:
nit: should we replace Metadata.Labels with Labels instead of adding? Only the latter should be used (?)